.github/workflows/full-service-tests.yml

```diff
@@ -32,59 +26,47 @@ jobs:
     - name: Checkout current code
       uses: actions/checkout@v3
 
-    - name: Set env
-      id: set-env-step
-      run: |
-        ENV=local
-        if ${{ github.event_name == 'workflow_dispatch' }}; then
-          ENV=${{ github.event.inputs.testEnv }}
-        fi
-        echo test-env=$ENV >> $GITHUB_OUTPUT
-
     - name: Set up JDK
       uses: actions/setup-java@v3
       with:
         distribution: 'temurin'
         java-version: 17
 
-    - name: Cache Gradle packages
-      uses: actions/cache@v3
-      with:
-        path: |
-          ~/.gradle/caches
-          ~/.gradle/wrapper
-        key: v1-${{ runner.os }}-gradle-${{ hashfiles('**/gradle-wrapper.properties') }}-${{ hashFiles('**/*.gradle') }}
-        restore-keys: v1-${{ runner.os }}-gradle-${{ hashfiles('**/gradle-wrapper.properties') }}
-
-    - name: Grant execute permission for gradlew
-      run: chmod +x gradlew
+    - name: Setup Gradle
+      uses: gradle/actions/setup-gradle@v3
 
-    - name: Get Vault token
-      id: vault-token-step
-      env:
-        VAULT_ADDR: https://clotho.broadinstitute.org:8200
-      run: |
-        VAULT_TOKEN=$(docker run --rm --cap-add IPC_LOCK \
-          -e "VAULT_ADDR=${VAULT_ADDR}" \
-          vault:1.1.0 \
-          vault write -field token \
-            auth/approle/login role_id=${{ secrets.VAULT_APPROLE_ROLE_ID }} \
-            secret_id=${{ secrets.VAULT_APPROLE_SECRET_ID }})
-        echo ::add-mask::$VAULT_TOKEN    
-        echo vault-token=$VAULT_TOKEN >> $GITHUB_OUTPUT
-
-    - name: Write config
-      id: config
-      uses: ./.github/actions/write-config
+    - name: Write credentials
+      id: write-credentials
+      uses: ./.github/actions/write-credentials
       with:
-        target: local
-        vault-token: ${{ steps.vault-token-step.outputs.vault-token }}
+        user-delegated-sa-b64: ${{ secrets.USER_DELEGATED_SA_DEV }}
+        buffer-client-sa-b64: ${{ secrets.BUFFER_CLIENT_SA_DEV }}
+        testrunner-sa-b64: ${{ secrets.TESTRUNNER_SA_DEV }}
+        testrunner-k8s-sa-b64: ${{ secrets.TESTRUNNER_K8S_SA_DEV }}
+        wsm-sa-b64: ${{ secrets.WSM_SA_DEV }}
+        janitor-sa-b64: ${{ secrets.JANITOR_SA_DEV }}
+        policy-client-sa-b64: ${{ secrets.POLICY_CLIENT_SA_DEV }}
+
+      # Turn on TPS for integration tests
+    - name: Write config
```

Comment:
Where was TPS enabled previously?

Reply:
Via the `write-config.sh` script invocation in the now dead `write-config` action. It might be worth it to universally enable TPS via a config change as I don't think there are any envs that don't have it enabled any more.